third_party/envtest/envtest.go

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Again, since this has nothing to do with envtest, I would suggest to move it to `pkg/envfuncs` package because it makes sense as a core part of the framework.